<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title>Brownie Points - Genetics and Pharmacology</title>
        <meta name="viewport" content="width=device-width">

        <!-- syntax highlighting CSS -->
        <link rel="stylesheet" href="/css/syntax.css">

        <!-- Custom CSS -->
        <link rel="stylesheet" href="/css/main.css">

		<!-- Set refresh timer for local development 
		<meta http-equiv="refresh" content="10" />-->
		
    </head>
    <body>

        <div class="container">
          <div class="site">

			<h1>Brownie Points - Genetics and Pharmacology</h1>
			<p class="meta">03 Dec 2013</p>

			<div class="post">
			<p>Headers are an outline of the Fundamentals 2, Exam 3 material provided by Dr. Brownie. I’ve rearranged the order to be a little more logical.</p>

<h1 id="genetics">GENETICS</h1>

<h3 id="risk-assessment-using-family-history">Risk assessment using family history</h3>
<ul>
  <li><strong>Autosomal Dominant (Ad)</strong>: 50% recurrence risk to offspring, males and females equally affected.</li>
  <li><strong>Autosomal Recessive (ar)</strong>: Parent of affected individual obligate carrier; 50% carrier risk to grandparents, aunts, and uncles of affected individual; transmission risk is 50%</li>
  <li><strong>Autosomal Recessive Risk Calculation</strong>: (Mom’s risk) *  (Dad’s risk) * (Transmission risk) </li>
  <li><strong>X-Linked Risk Calculation</strong>: (Mom’s risk) *  (Dad’s risk) * (Transmission risk) * (Gender risk) </li>
</ul>

<p><br /></p>

<h3 id="risk-assessment-using-hardy-weinberg">Risk assessment using Hardy-Weinberg</h3>
<ul>
  <li>Calculating the risk of carrier or affected status of a autosomal recessive or X-linked disease requires knowledge about both parents.</li>
  <li>If the carrier status of one of the parents cannot be determined by family history, you need to assume a frequency from population genetics</li>
  <li>Hardy-Weinberg Equilibrium: p + q = 1; (p+q)<sup>2</sup> = 1; p<sup>2</sup> + 2pq + q<sup>2</sup> = 1</li>
  <li><strong>Incidence rate == q<sup>2</sup></strong></li>
  <li>Take square root of incidence rate to get q</li>
  <li>Assume p is close enough to 1 that the difference is negligible</li>
  <li><strong>Carrier probability == 2q</strong></li>
</ul>

<p><br /></p>

<h3 id="carrier-risk-and-new-mutation-in-duchenne-muscular-dystrophy-dmd">Carrier risk and new mutation in Duchenne Muscular Dystrophy (DMD)</h3>
<ul>
  <li>DMD is inherited in an X-linked manner</li>
  <li><strong>maternal risk</strong> of transmission is 2/3</li>
  <li><strong>de novo</strong> mutation risk is 1/3</li>
  <li>Thus, maternal carrier risk calculations should be multiplied by 2/3 if it’s not known whether she’s a carrier.</li>
</ul>

<p><br /></p>

<h3 id="mechanisms-of-prader-willi-syndrome-and-angelman-syndrome">Mechanisms of Prader-Willi Syndrome and Angelman Syndrome</h3>
<ul>
  <li><strong>Prader-Willi Syndrome (PWS)</strong>: caused by absence of paternal 15q11-13</li>
  <li>PWS presents with short stature, cognitive and language deficits, hyperphagia, hypogonadicism</li>
  <li>PWS caused by paternal 15q11-13 deletion (70%), maternal uniparental disomy (30%), or imprinting center mutation (1%)</li>
  <li>Diagnosis: FISH, DNA methylation study</li>
  <li><strong>Angelman Syndrome (AS)</strong>: caused by an absence of maternal 15q11-13</li>
  <li>AS presents with severe developmental delay, wide-based gait, neurogenic laughter</li>
  <li>AS caused by maternal 15q11-13 deletion (68%), paternal uniparental disomy (7%), imprinting center mutation, <em>UBE3A</em> mutation, idiopathic</li>
  <li>Diagnosis: FISH, DNA methylation study, chromosome microarray</li>
  <li>Negative methylation study does not rule out AS due to other genetic etiologies</li>
</ul>

<p><br /></p>

<h3 id="clinical-indications-for-chromosome-microarray-analysis">Clinical indications for chromosome microarray analysis</h3>
<ul>
  <li>Microarray analysis is very effective in looking for deletions and duplications large regions such as chromosomes or even the entire genome. </li>
  <li>Congenital malformations</li>
  <li>Developmental delay</li>
  <li>Autism</li>
  <li>Unexplained stillbirth or neonatal death</li>
  <li>Recurrent miscarriage or infertility</li>
  <li>Advanced maternal age</li>
  <li>Positive family history of a chromosomal abnormality</li>
</ul>

<p><br /></p>

<h3 id="mitochondrial-disorder-etiologies">Mitochondrial disorder etiologies</h3>
<ul>
  <li><strong>Maternal cytoplasmic inheritance</strong> - essentially all mitochondria come from the mother</li>
  <li><strong>Mitochondrial heteroplasmy</strong> - not all mitochondria have the same mtDNA</li>
  <li><strong>Replicative segregation</strong> - both aberrant and non-aberrant mitochondrial genomes are replicating, cause of phenotype threshold</li>
  <li><strong>Graded phenotype</strong> - phenotype is determined by <em>amount</em> and <em>distribution</em> of aberrant genome</li>
  <li>Most common mitochondrial genetic diseases: <strong>MELAS</strong>, <strong>MERRF</strong>, <strong>NARP</strong>, <strong>LHON</strong></li>
</ul>

<p><br /></p>

<h3 id="role-of-sonic-hedgehog-shh">Role of Sonic Hedgehog (Shh)</h3>
<ul>
  <li>Sonic hedgehog (Shh) is a key paracrine factor implicated in embryogenesis</li>
  <li>Critical role we should know is its orchestration of anterior cleavage of the prosencephalon</li>
  <li>Alterations in Shh homeostasis leads to craniofacial defects, role in fetal alcohol syndrome (FAS)</li>
</ul>

<p><br /></p>

<h3 id="teratogen-risks">Teratogen risks</h3>
<ul>
  <li><strong>Major teratogen risks</strong>:</li>
  <li><em>Thalidomide</em>: leads to limb deformities, tetraphocomelia</li>
  <li><em>ETOH</em>: causes fetal alcohol syndrome (FAS) via decrease in [Shh] and [PAX]</li>
  <li><em>cis-retinoic acid</em>: leads to craniofacial defects</li>
  <li><em>valproic acid</em>: leads to neural tube defects (NTD)</li>
</ul>

<p><br /></p>

<h3 id="trinucleotide-repeats-in-huntington-disease">Trinucleotide repeats in Huntington disease</h3>
<ul>
  <li>Huntington disease is inherited in an autosomal dominant manner caused by a trinucleotide (CAG)<sub>n</sub> repeat in the <em>HD</em> gene.</li>
  <li>Huntington disease is characterized by paternal anticipation, thus daughters of affected fathers typically have an earlier onset of disease</li>
  <li>TNR counts to know: less than 26 is normal, 27-35 intermediate, 36-39 reduced penetrance, greater than 40: disease</li>
</ul>

<p><br /></p>

<h3 id="anticipation">Anticipation</h3>
<ul>
  <li>Trinucleotide repeats can be expected to increase every generation causing either earlier onset or worsened phenotype or both</li>
  <li>Maternal anticipation: <strong>sons are more affected</strong> (e.g. myotonic dystrophy I, Fragile X syndrome)</li>
  <li>Paternal anticipation: <strong>daughters are more affected</strong> (e.g. Huntington disease)</li>
</ul>

<p><br />	</p>

<h3 id="williams-syndrome">Williams syndrome</h3>
<ul>
  <li>Williams Syndrome is a developmental disorder caused by a multigenic deletion at 7q11-23</li>
  <li>Involvement of the <em>ELN</em> gene, which encodes elastin, is responsible for the cardiovascular pathology.</li>
  <li>Patients retain significant language functions but with low IQs and need for constant care.</li>
  <li>Diagnosis: FISH</li>
</ul>

<p><br /></p>

<h3 id="detection-of-microdeletion-syndrome">Detection of microdeletion syndrome</h3>
<ul>
  <li>For microdeletion syndromes, karyotyping alone is not high enough resolution to detect deletion</li>
  <li>Thus, you need to use fluoresence in-situ hybridization (FISH); pathology usually indicated by presence of two control markers but only one target marker</li>
</ul>

<p><br /></p>

<h3 id="x-inactivation">X inactivation</h3>
<ul>
  <li>Principle: <strong>single active X</strong></li>
  <li>Occurs ~2 weeks in to development at approximately 100 cell stage</li>
  <li>Mechanism: <em>XIST</em> on Xq13 of to-be inactivated X is expressed, RNA product remains in nucleus and coats the X chromosome to suppress transcription</li>
  <li><strong>Random</strong>: either X<sup>mom</sup> or X<sup>dad</sup> is inactivated</li>
  <li><strong>Clonal</strong>: daughter cells’ inactivated X depends on parent cell</li>
  <li><strong>Incomplete</strong>: There is some transcription from the inactivated X, mostly in PAR of Xp</li>
  <li><strong>Exceptions!</strong></li>
  <li>Structurally Abnormal X Chromosome - selective inactivation of the abnormal X</li>
  <li>Balanced X-autosome Translocation - selective inactivation of the normal X</li>
  <li>Unbalanced X-autosome Translocation - selective inactivation of the abnormal X</li>
</ul>

<p><br /></p>

<h3 id="risk-assessment-in-balanced-translocation">Risk assessment in balanced translocation</h3>
<ul>
  <li>A balanced rearrangement leads to a normal complement of DNA with no net gain or loss.</li>
</ul>

<p><br /></p>

<h3 id="unbalanced-rearrangements">Unbalanced rearrangements</h3>
<ul>
  <li>An unbalanced rearrangement leads to an abnormal complement of DNA with net gain or loss.</li>
  <li>Types: <strong>deletion</strong>, <strong>duplication</strong>, <strong>insertion</strong>, <strong>inversions</strong>, <strong>isochromosome</strong>, <strong>ring chromosome</strong></li>
  <li>Chromosomal inversions: paracentric vs. pericentric. Unbalanced paracentric are non-viable.</li>
</ul>

<p><br /></p>

<h3 id="translocation-down-syndrome">Translocation Down syndrome</h3>
<ul>
  <li>95% of Down syndrome is caused by Trisomy 21</li>
  <li>3% due to Robertsonian translocation (t14:21)</li>
  <li>Robertsonian translocations are the fusion of two acrocentric chromosomes (13, 14, 15, 21, 22), resulting in a karotype with 45 chromosomes</li>
  <li>50% of cases have a balanced carrier parent.</li>
  <li>Risk of Down syndrome if mother is carrier 10-15%, if father is carrier, &lt; 1%.</li>
</ul>

<p><br /></p>

<h3 id="meiosis">Meiosis</h3>
<ul>
  <li>Key process for gametogenesis, reduces n to 1/2 n</li>
  <li>Meiosis 1: ‘reduction division’ - splits mom and dad DNA. </li>
  <li>Meiosis 2: identical to mitosis, but resulting in cells with hopefully only 23 chromosomes</li>
  <li>P1, M1, A1, T1, no DNA replication, P2, M2, A2, T2</li>
  <li>Steps in Prophase 1 (P1): LZPDD “Led Zeppelin Plays Dancing Days”: Leptotene, zygotene, pachytene, diplotene, diakinesis</li>
  <li>Crossing over occurs during P1.pachytene</li>
  <li><strong>Errors of meiosis</strong>: Most important: nondisjunction, the failure of separation of homologous chromosomes (M1) or sister chromatids (M2, mitosis)</li>
</ul>

<p><br /></p>

<h3 id="gender-differences-in-meiosis">Gender Differences in Meiosis</h3>
<ul>
  <li><strong>Male meiosis</strong>: commences at puberty, takes about 60-65 days to complete. Each cycle results in 4 spermatids. Hundreds of thousands made daily to weekly.</li>
  <li><strong>Female meiosis</strong> begins at 3 month of embryonic life; zygotene reached by 4th month. Diplotene complete by 7th month. Process is arrested at the 9th month in a new state, <strong>dictyotene</strong>. 9-13 years pass, menarche arrives. As oocyte develops, meiosis 1 is resumed. First polar body formed by pair of homologous chromosomes. Metaphase II occurs around the time of ovulation. If fertilization occurs, meiosis II can complete, resulting in 2nd polar body formation </li>
  <li>1st polar body contents: homologous pair of sister chromatids</li>
  <li>2nd polar body contents: sister chromatid to fertilized egg</li>
  <li>Retrieve and sequence polar bodies for advanced diagnostics and therapeutics</li>
</ul>

<p><br /></p>

<h3 id="genetic-recombination">Genetic recombination</h3>
<ul>
  <li>Occurs in pachytene of prophase I of meiosis; “mixing of the maternal and paternal”</li>
  <li>Only one sister chromatid is involved in each event.</li>
  <li>Rate of female recombination &gt; rate of male recombination</li>
  <li>Decreases near centromere, increases near telomere such that male telomeric recombination rate is greater than female.</li>
  <li>Recombination can cause: <strong>insertions</strong>, <strong>deletions</strong>, or <strong>inversions</strong>.</li>
</ul>

<p><br /></p>

<h3 id="linkage-analysis">Linkage analysis</h3>
<ul>
  <li>DNA linkage analysis uses polymorphic inherited sequences to track nearby sequences of interest from one person to another within the pedigree.  </li>
  <li>This technique could be used to confirm transmission of a disease-related gene from a parent to, for example, a fetus if the purpose is for prenatal diagnosis.  </li>
  <li>Testing for rare, disease-related alleles or mutations is better done by direct analysis using oligonucleotides or sequencing methods.</li>
  <li>From class notes: Increased distance between loci lead to increased likelihood of crossing-over</li>
  <li>1% chance of recombination = 1 cM ~ 1x10<sup>6</sup> BP</li>
  <li>Maximum likelihood = 50%</li>
</ul>

<p><br /></p>

<h3 id="recognition-of-sex-chromosome-abnormalities-in-the-newborn">Recognition of sex chromosome abnormalities in the newborn</h3>
<ul>
  <li><strong>Turner Syndrome</strong> (typically 45,X): short stature, webbed neck, loose nuchal skin, Clinical features: short stature, webbed neck, leads to cardiac and renal defects, infertility and amenorrhea. <em>Mechanism</em>: paternal nondisjunction, maternal nondijunction, mosaicism, isochromosome (2 q arms on karyotype)</li>
  <li><strong>Klinefelter Syndrome</strong> (typically 47,XXY): tall, phenotypic normal til puberty, testicular hypoplasia, gynecomastia, infertility. <em>Mechanism</em>: paternal or maternal nondisjunction.</li>
  <li><strong>47,XYY</strong>:  tall, phenotypically normal, gonadonormal, fertile, learning / behavioral problems. <em>Mechanism</em>: paternal nondisjunction in MII.</li>
  <li><strong>47,XXX</strong>: +/- hypotonia, phenotypic normal, cognitive and social deficits, normal sexual development with premature ovarian involution and menopause. <em>Mechanism</em>: maternal nondisjunction in M1</li>
</ul>

<p><br /></p>

<h3 id="sex-chromosome-abnormalities-and-fertility">Sex chromosome abnormalities and fertility</h3>
<ul>
  <li>Turner syndrome: infertility</li>
  <li>Klinefelter syndrome: infertility, hypogonadism</li>
  <li>47,XYY: fertile, gonadonormal.</li>
  <li>47,XXX: usually fertile with early menopause due to accelerated ovarian involution</li>
  <li>Also of note: <em>DAZ<sup>-</sup></em> located at Yq11 causes azoospermia and oligospermia, accounts for 10-15% of male infertility</li>
</ul>

<p><br /></p>

<h3 id="indications-for-prenatal-diagnosis">Indications for Prenatal Diagnosis</h3>
<ul>
  <li>Advanced maternal age</li>
  <li>Positive serum marker screen in the first or second trimester</li>
  <li>Previous child with chromosome abnormality</li>
  <li>One parent is the carrier of a balanced translocation</li>
  <li>Previous fetus or child with a neural tube defect</li>
  <li>Both parents carriers for prenatally diagnosable autosomal recessive disorder</li>
  <li>One parent affected with prenatally diagnosable autosomal dominant disorder</li>
  <li>Mother is carrier of an X-linked disorder</li>
  <li>Fetal anomalies detected on ultrasound</li>
  <li>Anxiety, in the absence of above conditions </li>
</ul>

<p><br /></p>

<h3 id="chorionic-villus-sampling-and-amniocentesis">Chorionic villus sampling and amniocentesis</h3>
<ul>
  <li>Amniocentesis: Effective, safe (loss at 1%). Performed at 16-18 weeks. 7d - 4wk turn around time. Pseudomosaicism and maternal cell contamination are limitations. </li>
  <li>Chorionic villus sampling (CVS): available earlier than amniocentesis but slightly more risky. Look for fetal contribution to the placenta, analyze those cells. Transvaginal or transabdominal approach with no significant change in risk.  </li>
</ul>

<p><br /></p>

<h3 id="uses-of-msafp">Uses of MSAFP</h3>
<ul>
  <li>MSAFP is used as a neonatal screening test.</li>
  <li>Abnormalities involving leakage of fetal CSF or blood will increase the AFP concentration in amniotic fluid as well as in the mother’s blood.  </li>
  <li>Normal distribution centered at 1, measured in ‘multiples of median’.</li>
  <li>Down &lt; Normal &lt; Spina Bifida &lt; Anencephaly</li>
  <li>AFP is to the fetus what serum albumin is to the child postnatally.  </li>
  <li>Its concentration his highest in fetal blood, so bleeding elevates AFP concentration in amniotic fluid, as does the leakage of cerebrospinal fluid in amniotic fluid (hence the effects of open neural tube defects).  </li>
  <li>Normal newborns have very high concentrations of AFP in serum, but this rapidly decreases and is replaced by serum albumin produced in the newborn’s liver.</li>
  <li>Composite score of age + AFP + other fetal proteins allows for better detection</li>
</ul>

<p><br /></p>
<hr />

<p><br />
## PHARMACOLOGY
<br /></p>

<h3 id="drug-receptor-definition">Drug receptor definition</h3>
<ul>
  <li>a specific macromolecular constituent of the biological system with which a drug interacts to produce an effect</li>
</ul>

<p><br /></p>

<h3 id="representative-types-of-receptors">Representative types of receptors</h3>
<ul>
  <li>Ligand-gated ion channel</li>
  <li>Membrane-embedded enzyme</li>
  <li>G protein-coupled receptor</li>
  <li>Transcription factor</li>
</ul>

<p><br /></p>

<h3 id="drug-names">Drug names:</h3>
<ul>
  <li>chemical</li>
  <li>generic </li>
  <li>proprietary (trade)</li>
</ul>

<p><br /></p>

<h3 id="multiple-receptors-for-drugs-eg-lidocaine-tamoxifenselective-toxicity">Multiple receptors for drugs, e.g., lidocaine, tamoxifen/selective toxicity</h3>
<p>### Binding forces in the drug-receptor complex
- Irreversible:  covalent
- Reversible:  ionic, hydrogen, dipole-dipole, van der Waals, hydrophobic</p>

<p><br /></p>

<h3 id="sp3-hybridization-centers-of-chirality-stereoselectivity-of-receptors-racemic-mixtures">SP3 hybridization, centers of chirality, stereoselectivity of receptors, racemic mixtures</h3>
<p>### Drug-receptor interactions, the law of mass action, the occupancy assumption:  E/Emax = X/Kd + X
### Semi-log plots:  E/Emax vs log of concentration or dose
### Spare receptors
### Concentration-effect relationship in isolated systems:  affinity and intrinsic activity
### Dose-effect relationship in intact systems:  potency and efficacy
### Drug interactions:  summation and potentiation
### Drug interactions:  antagonism
- Definition of an agonist 
- Definition of an antagonist</p>

<p><br /></p>

<h3 id="types-of-antagonism">Types of antagonism</h3>
<ul>
  <li>Physiological</li>
  <li>Pharmacological</li>
  <li>Surmountable </li>
  <li>Insurmountable</li>
</ul>

<p><br /></p>

<h3 id="partial-agonists-agonistantagonists">Partial agonists (agonist/antagonists)</h3>
<p>### Constitutively active systems:  inverse agonism
### Therapeutic index, multiple indices
### Dose-response relationships:  graded vs quantal
### Weak acids and bases:  role of ionization in lipid solubility/renal excretion 
### Binding of drugs to plasma proteins
### Blood-brain barrier/placental barrier 
### Routes of administration
- Enteral:  	
- By mouth (PO) and the first pass effect
- Mucous membranes 
- Intra-arterial
- Intra-venous
- Sub-cutaneous
- Intramuscular
- Epidural
- Intrathecal
- Intraosseus
- Transdermal
- Respiratory tract</p>

<p><br /></p>

<h3 id="comparison-of-routes-re-absorption-and-hazards">Comparison of routes re absorption and hazards</h3>
<p>### Compliance/adherence 
### Drug tolerance/cross tolerance
### Physical dependence/cross dependence
### Addiction
### Homeostatic theory of tolerance and physical dependence
### Script doctors/physician addicts
### Plasma drug kinetics
### Plasma half time for drugs
### Drug body burden
### Excretion rate constants for drugs
### Properties of mixed-function oxidases (MFO)
### Phase I and Phase II of drug metabolism
### Reactions catalyzed by MFOs</p>

			</div>
			<br>
			<hr>
            <div class="footer">
			<h6><p><a href="/">&#60; Back</a></p></h6>
			</div>
           	<div class="contact">
                <p>
					<a href="http://bquarant.github.io/">bquarant.github.io</a> | 2014 
               </p>
              </div>
          </div>
        </div>

<!-- Track traffic on gaug.es-->
		<script type="text/javascript">
		  var _gauges = _gauges || [];
		  (function() {
		    var t   = document.createElement('script');
		    t.type  = 'text/javascript';
		    t.async = true;
		    t.id    = 'gauges-tracker';
		    t.setAttribute('data-site-id', '51f19a5df5a1f51079000024');
		    t.src = '//secure.gaug.es/track.js';
		    var s = document.getElementsByTagName('script')[0];
		    s.parentNode.insertBefore(t, s);
		  })();
		</script>
    </body>
</html>